packages/web-components/fast-components/src/carousel/carousel.styles.ts

```diff
@@ -0,0 +1,123 @@
+import { css } from "@microsoft/fast-element";
+import {
+    accentFillRestBehavior,
+    neutralDividerRestBehavior,
+    neutralForegroundRestBehavior,
+    neutralFocusBehavior,
+} from "../styles/recipes";
+import { focusVisible } from "@microsoft/fast-foundation";
+
+export const CarouselStyles = css`
+    :host {
+        box-sizing: border-box;
+        font-family: var(--body-font);
+        font-size: var(--type-ramp-minus-1-font-size);
```

Comment:
you should interpolate this `var` string: `color: ${neutralForegroundrest.var};`